docs/howto/smart_contract_upload.md

```diff
@@ -0,0 +1,138 @@
+# Smart Contract Upload
+
+This tutorial covers how to create a circuit on a splinter network between 
+two nodes and upload a smart contract via the circuit.
+
+## Prerequisites
+
+* A splinter network with at least two nodes running. The tutorial will refer
+  to the two nodes as "alpha" and "beta" respectively
+* The scabbard command line tool installed with experimental features enabled
+* The splinter command line tool installed with experimental features enabled
```

Comment:
It would be nice to link to the info on enabling experimental features in the Splinter README. Here's how to link to the "Building Splinter" section in the README:

`[with experimental features enabled](https://github.com/Cargill/splinter/blob/master/README.md#building-splinter)`